code/modules/projectiles/ammunition.dm

```diff
@@ -122,14 +122,12 @@ They're all essentially identical when it comes to getting the job done.
 
 //This will attempt to place the ammo in the user's hand if possible.
 /obj/item/ammo_magazine/proc/create_handful(mob/user, transfer_amount)
-	var/R
 	if(current_rounds <= 0)
 		return
-
+	var/R
 	var/obj/item/ammo_magazine/handful/new_handful = new /obj/item/ammo_magazine/handful()
-	var/MR = (caliber in list(CALIBER_12G, CALIBER_762x54, CALIBER_410)) ? 5 : 8
-	R = transfer_amount ? min(current_rounds, transfer_amount) : min(current_rounds, MR)
-	new_handful.generate_handful(default_ammo, caliber, MR, R, gun_type)
+	R = transfer_amount ? min(current_rounds, transfer_amount) : min(current_rounds, initial(default_ammo.handful_amount))
```

Comment:
You can just directly do var/R = transfer_amount ? whatever

On a side note R is a horrid variable name